_rules/device-motion-disabled-c249d5.md

```diff
@@ -30,12 +30,14 @@ The rule applies to an [HTML document][] with an associated [Window object][] th
 
 ## Expectation 1
 
-For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] to prevent any results of the event.
+For each registered [device orientation event][device orientation] or [device motion event][device motion] in the test target there exists a set of one or more [instruments][instrument] to prevent any results of the event within a 10 minute time span.
```

Comment:
There might need to precise to what "within 10 minutes" applies.
* Activating the instrument "disable" the event forever, where "disable" means "no effect during 10 minutes". (real case)
* Activating the instrument "disable" the event for the next 10 minutes. (this interpretation is plausible with the text)
* Activating the instrument can take up to 10 minutes to react, after that the event is "disabled" forever. (this interpretation is a bit far fetched)

Reply:
@Jym77 I've updated the expectation. Can you please review again?